testsuite/regressiontests/ticket-77.sli

```diff
@@ -107,6 +115,10 @@ M_ERROR setverbosity
                           % plastic synapses.
 
       /initial_weight 10.0 def
+      model higher_weight_list exch MemberQ
+      {
+        /initial_weight 80.0 def
+      } if
```

Comment:
@Silmathoron This is fine, but you could integrate it with the `/extra_params` dictionary by adding to that dictionary for each of the affected models 
```
   /aeif_psc_exp_ps << /initial_weight 80.0 >>
```
with the corresponding model names. Then you can put in the definition of `/syn_spec`
```
/weight 
   extra_params model known
   {
      extra_params model get /initial_weight known
      {
         extra_params model get /initial_weight get
      }
      {
         initial_weight 
      } ifelse
   }
   {
      initial_weight
   } ifelse
```
similar to  the setup for `/receptor_type`.